auth-web/tests/unit/test-utils/test-data/affiliations.ts

```diff
@@ -15,68 +24,78 @@ export const businesses = [
   },
   // BEN Name Request
   {
-    identifier: 'NR 4045466',
-    legalType: 'NR',
+    businessIdentifier: undefined,
+    nrNumber: 'NR 4045466',
+    corpType: { code: CorpTypes.NAME_REQUEST },
     name: 'BEN NAME REQUEST LIMITED',
     nameRequest: {
-      legalType: 'BEN',
-      names: [{ 'name': 'BEN NAME REQUEST LIMITED' }],
+      legalType: CorpTypes.BENEFIT_COMPANY,
+      names:
+      [{
+        name: 'BEN NAME REQUEST LIMITED',
+        decision_text: 'Decision text',
+        name_type_cd: 'Name type code',
+        designation: 'Designation',
+        state: 'State'
+      }],
       nrNumber: 'NR 4045466',
       state: 'APPROVED',
-      expirationDate: '2022-11-02T19:42:13+00:00'
+      expirationDate: new Date('2022-11-02T19:42:13+00:00')
```

Comment:
new Date, will cause problems when the unit tests run in UTC but your local machine runs in PST

Reply:
added a function convert date to UTC